config/management/src/secure_backend.rs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_config::config::{OnDiskStorageConfig, SafetyRulesBackend, VaultConfig};
+use libra_secure_storage::Storage;
+use std::{collections::HashMap, convert::Into, path::PathBuf};
+
+/// SecureBackend is a parameter that is stored as a colon separated key/value pairs. The only
+/// expected key is backend which defines which of the SecureBackends the parameters refer to.
+/// Some backends require parameters others do not, so that requires a conversion into the
+/// SafetyRulesBackend type to parse.
```

Comment:
Could we not explicitly update the comment to outline what parameters are acceptable? I know this is enforced by the tests, but a simple comment is quicker to read and might save someone a lot of time if they want to use this.
For example:
- For in_memory you only need to state: "backend=memory". No other key/value pairs are required.
- For on_disk, you also need to state the path to the on-disk file, e.g."backend=on_disk ; path=path_to_file".
...

Reply:
Good call! I plan to do that when I integrate it into structopt

Reply:
Sounds good!